### DISCUSSION TITLE
Multiple profile support

### DISCUSSION THREAD

**Discussion by rediffusion** [xiaoxiaoflood/firefox-scripts#192] [Requests]
Hi, dear @xiaoxiaoflood❗

> The system

Release: Windows 11 Pro
Version: 21H2
Install Date: 14.10.2022
Build OS: 22621.674
Communication: Windows Feature Experience Pack 1000.22634.1000.0

> Browser

▪️Firefox 105.0.3 (64 bit) `I use it a lot` 🤡

---
 Is there multiple Profiles support? Cuz I have 6 more. I've done the same manipulation but nothing works.

All the best, see you soon. 😋

**Comment by megamorphg** [comment]
Works fine for me on multiple profiles Firefox on both Firefox Release and Firefox Developer. Just make sure to follow the same instructions and update the app directory.
